sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/implementation/models/RecordingContentType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
In Java, if the model is under `implementation` folder, then it will not be exposed publically in the SDK.
Adding the script below to the code generation settings in the swagger README will move the models out of the implementation/models folder and into the models folder when you re-generate.

Linking SMS swagger README as an example: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-sms/swagger/README.md
```
custom-types: <ModelName>
custom-types-subpackage: models
```